CONTRIBUTING.md

```diff
@@ -1,62 +1,157 @@
 
-  - The `title` of the new SVG.
-  - A `hex` color value that matches the brand or logo's main accent color. (Without the `#` pound symbol.)
-  - The `source` URL of the logo being used.
+We welcome icon requests. However, before you submit a new issue please make sure the icon:
 
-Here is the object for the GitHub logo as an example:
+* Is of a _popular_ brand.
+* Has not already been requested.
+* Can be represented in a monochromatic format.
 
-```json
-{
-  "title": "GitHub",
-  "hex": "181717",
-  "source": "https://github.com/logos"
-},
-```
+When submitting a request for a new icon, please submit the issue with the following information:
+
+* **Title:** Include the brand name. Example: `Request: GitHub Icon`
+* **Body:** Include links to official sources for the brand's icon and official colors (e.g. media kits, brand guidelines, SVG files etc.)
+
+When submitting a request to update an existing icon, please submit the issue with the following information:
+
+* **Title:** Include the brand name. Examples: `Update: GitHub Color`, `Update: GitHub Icon`
+* **Body:** Include what needs updating. The color? The icon? Include links to official sources for the updated icon or color (e.g. media kits, brand guidelines, SVG files etc.)
+
+## Adding or Updating An Icon
+
+### 1. Identify Official Logos and Colors
+
+Most of the icons and brand colors on SimpleIcons have been derived from official sources. Using official sources helps ensure that the icons and colors in SimpleIcons accurately match the brand they represent. Thankfully, this is usually a simple process as organizations often provide brand guides and high quality versions of their logo for download.
+
+Official high quality brand logos and brand colors can usually be found in the following locations:
+
+1. About pages, Press pages, Media Kits, and Brand Guidelines.
+1. Website headers
+1. Favicons
+1. Wikimedia (which should provide a source)
+
+Working with an SVG version of the logo is best. In the absence of an SVG version, other vector filetypes may work as well such as EPS. In the absence of vector logos, a vector can be created from a high quality rasterized image, however this is much more labor intensive.
+
+For color, the brand's primary web color should be used. Simple Icons stores brand colors in the standard 6 character hexadecimal format.
+
+### 2. Extract The Icon From The Logo
+
+There are many different tools for editing SVG files, some options include:
+
+| Name | Description | Platform | Price | Wikipedia |
+| :---- | :---- | :----: | :----: | ----: |
+| [Inkscape](https://inkscape.org/en/)| Vector Graphics Editor | Win, Mac, Linux | Free | [Wiki](https://en.wikipedia.org/wiki/Inkscape) |
+| [Adobe Illustrator](https://www.adobe.com/products/illustrator.html) | Vector Graphics Editor | Win, Mac | $ - $$$ | [Wiki](https://en.wikipedia.org/wiki/Adobe_Illustrator) |
+| [IcoMoon](https://icomoon.io/) | Icon Editing/Management Tool | Online | Free | -- |
```

Comment:
I've got two remarks regarding this table:
- I would personally write `Windows` instead of `Win`
- I'm not sure what value the wiki link adds on top of the tools homepage, so I would leave it out